src/client/stylesheets/style.less

```
@@ -414,6 +414,10 @@ hr.wide {
 	}
 }
 
+.next, .previous > a {
```

This applies differently for next and for previous: you target elements with the .next class as well as <a> elements inside a parent with the class .previous (but not the element with the .previous class).

Using a percentage here can be tricky in some situations; instead let's use `1em` (or 0.7em, or whatever number you feel is right, but as an `em` value).

I would also like to make sure this applies only where we want it to, so altogether I suggest something like this: 
```suggestion
#pageWithPagination {
  .next, .previous {
    > a {
      margin-bottom: 1em;
    }
  }
}
```

---

Perfect! I'll do some testing to see if it only applies to those exact buttons